<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LedOper</class>
 <widget class="QWidget" name="LedOper">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="lab_curr_time">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>20</y>
     <width>401</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lab_led">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>70</y>
     <width>101</width>
     <height>211</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_strategy">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>140</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>to Manunal mode</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_led_oper">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>220</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>open</string>
   </property>
  </widget>
  <widget class="QLabel" name="lab_curr_value">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>59</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lab_cv">
   <property name="geometry">
    <rect>
     <x>163</x>
     <y>60</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>value</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
